skimage/util/dtype.py

```diff
@@ -134,165 +138,162 @@ def compare(x, y, kind='u'):
         return np.dtype(kind + str(s))
 
     def _scale(a, n, m, copy=True):
```

Comment:
Could the variables have more significant names? I think `a`, `b`, `m` and `n` do not express that much.